packages/block-library/src/latest-posts/edit.js

```diff
@@ -522,6 +523,16 @@ class LatestPostsEdit extends Component {
 	}
 }
 
+function LatestPostBlockWrapper( props ) {
```

Comment:
Ideally, we refactor the whole block to be a functional component, but this is better done on a separate PR. This should be enough for the moment.